spec/controllers/api/scores_controller_spec.rb

```diff
@@ -104,4 +104,26 @@
       expect(Score.count).to eq score_count
     end
   end
+
+  describe 'GET user_scores' do
+    it 'should return the scores related to the user' do
+      @user = create(:user, name: 'User', email: 'user7@email.com', password: 'userpass')
+
+      sign_in(@user, scope: :user)
+
+      @score1 = create(:score, user: @user, total_score: 79, played_at: '2021-05-20')
+      @score2 = create(:score, user: @user, total_score: 59, played_at: '2021-05-25')
```

Comment:
Why did you choose to define another user? You already have user1 and user2.

Reply:
I've changed this to use the user1 defined in before each block.